selfdrive/car/card.py

```diff
@@ -167,6 +174,14 @@ def __init__(self, CI=None, RI=None) -> None:
     self.params.put_nonblocking("CarParamsCache", cp_bytes)
     self.params.put_nonblocking("CarParamsPersistent", cp_bytes)
 
+    # Write CarParamsSP for controls
+    # convert to pycapnp representation for caching and logging
+    self.CP_SP_capnp = convert_to_capnp(self.CP_SP)
+    cp_sp_bytes = self.CP_SP_capnp.to_bytes()
```

Comment:
huh? why don't I need to do this when I store the ActiveBundle on the model selector? it's confusing me